rethinkdb/content.md

```diff
@@ -0,0 +1,46 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance allowing connections on all interfaces
+
+```bash
+docker run --name some-rethink -d rethinkdb --bind all
```

Comment:
Moreover, I think I should probably add a volume to the arg, since otherwise you're just running an ephemeral DB, which is somewhat useless.